tests/integration/nginx_test.go

```diff
@@ -1,19 +1,21 @@
+//go:build integration
 // +build integration
```

Comment:
Can we remove the older comment=

Reply:
Done in [#a5f5e48](https://github.com/newrelic/nri-nginx/pull/163/commits/a5f5e484e5ea45168a2f533ef728a4a2bb236cdc)